src/test/java/com/loofah/graph/api/queries/CategoryQueryTest.java

```diff
@@ -0,0 +1,45 @@
+package com.loofah.graph.api.queries;
+
+import com.loofah.graph.api.models.database.Category;
+import com.loofah.graph.api.repositories.CategoryRepository;
+import graphql.schema.DataFetchingEnvironment;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Optional;
+
+import static com.loofah.graph.api.helpers.TestHelpers.CATEGORY_ID;
+import static com.loofah.graph.api.helpers.TestHelpers.getDefaultCategoryBuilder;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CategoryQueryTest {
+
+    @Mock
+    private CategoryRepository categoryRepository;
+
+    @Mock
+    private DataFetchingEnvironment dataFetchingEnvironment;
+
+    private CategoryQuery categoryQuery;
+
+    @Before
+    public void setUp() {
+        categoryQuery = new CategoryQuery(categoryRepository);
+    }
+
+    @Test
+    public void get_findsSkillsFromRepositoryWithGivenId() {
```

Comment:
[nitpick] name of test should be "findCategory" rather than "findSkill"

Reply:
good spot - updated